### ISSUE TITLE
[Android] Error Getting FCM Token

### ISSUE THREAD

**Issue by edTheGuy00** [OneSignal/OneSignal-Flutter-SDK#228]
Notification on Android completely stopped working since upgrading to the latest version `2.3.4`. I have tried downgrading versions with no luck, I also found similar issues [here](https://github.com/OneSignal/OneSignal-Android-SDK/issues/542)  and [here](https://github.com/OneSignal/OneSignal-Cordova-SDK/issues/451) from a few years ago, I tried completely removing `firebase_auth` since it was in my project, as well as changing Google play version but no luck. 

As mentioned, everything was working perfectly fine before `2.3.4`.

Debug app is running on a  ONEPLUS A5010. And it appears all notifications stopped working on all devices that have installed the latest version of the app as this issue went unnoticed.
 
The error I keep on seeing on App Initialisation is:

```
E/FirebaseInstanceId(30415): Token retrieval failed: INVALID_SENDER
E/OneSignal(30415): Error Getting FCM Token
E/OneSignal(30415): java.io.IOException: INVALID_SENDER
E/OneSignal(30415): 	at com.google.firebase.iid.zzr.zza(Unknown Source:66)
E/OneSignal(30415): 	at com.google.firebase.iid.zzr.zza(Unknown Source:79)
E/OneSignal(30415): 	at com.google.firebase.iid.zzu.then(Unknown Source:4)
E/OneSignal(30415): 	at com.google.android.gms.tasks.zzd.run(Unknown Source:5)
E/OneSignal(30415): 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
E/OneSignal(30415): 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
E/OneSignal(30415): 	at java.lang.Thread.run(Thread.java:764)
```

**Comment by jkasten2** [comment]
@edTheGuy00 The stacktrace doesn't include `com.onesignal.PushRegistratorFCM.getToken` as the other tickets you have pointed to.

Since only firebase / google cases are in your stacktrace this could be getting pulled from the  `google-services.json` file in your project. This file isn't required to use OneSignal.

**Comment by edTheGuy00** [comment]
@jkasten2 We need `google-services.json` as we are using a couple of Firebase services, but  I think I found the issue. The previous developer had used his personal Firebase project for the OneSignal app, and recently deactivated his account (coincidentally around the same time I upgraded versions). I've already sent a message for support with changing the Firebase Server Key and Firebase Sender ID. 
<img width="533" alt="Screen Shot 2563-04-08 at 14 15 05" src="https://user-images.githubusercontent.com/16095735/78758962-1d865c00-79a9-11ea-90c8-9a043da0c781.png">
